src/models/IDatabaseService.ts

```diff
@@ -15,8 +15,14 @@ export type IDatabaseService = {
     stop: () => Promise<void>,
 
     /**
-     * Removes the user whose id is userid from the database.
+     * Soft-deletes the user whose id is userid from the database (sets a flag to true).
      * @param userid
      */
     delete: (userid: Snowflake) => Promise<void>,
+
+    /**
+     * Undo the soft-delete of the user whose id is userid (sets a flag to false).
+     *
+     */
+    undelete: (userid: Snowflake) => Promise<void>,
```

Comment:
Should I change to something more "english" ? Because "undelete" is weird, so maybe "undoDelete" would be more meaningful ?